packages/main/src/Carousel.js

```diff
@@ -0,0 +1,232 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import {
+	isLeft,
+	isRight,
+	isDown,
+	isUp,
+} from "@ui5/webcomponents-base/dist/events/PseudoEvents.js";
+import {
+	fetchI18nBundle,
+	getI18nBundle,
+} from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import ScrollEnablement from "@ui5/webcomponents-base/dist/delegate/ScrollEnablement.js";
+import {
+	CAROUSEL_OF_TEXT,
+} from "./generated/i18n/i18n-defaults.js";
+import CarouselArrowsPlacement from "./types/CarouselArrowsPlacement.js";
+import CarouselTemplate from "./generated/templates/CarouselTemplate.lit.js";
+import { isMobile } from "@ui5/webcomponents-base/dist/Device.js";
+
+// Styles
+import CarouselCss from "./generated/themes/Carousel.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-carousel",
+	properties: /** @lends sap.ui.webcomponents.main.Carousel.prototype */ {
+		/**
+		 * If set to true the navigation is hidden
```

Comment:
it would be cool to have the "loop/cycling" property as well, but it might be implemented later as well